examples/java-gradle/shared-module/module-b/cdktf.json

```diff
@@ -0,0 +1,11 @@
+{
+  "language": "java",
+  "app": "../gradlew -q :module-b:run",
+  "output": "./",
```

Comment:
Maybe add a comment about why you're using a non-default output. It took me a minute to release why this is necessary.

Reply:
Where would be the best place to address this? I could potentially add it to the README since this is a json file.
Thoughts?

Reply:
The README is probably a good spot. Could also add as a `//` property above the output property.